<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDisplayDlg</class>
 <widget class="QDialog" name="CustomerDisplayDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CustomerDisplayDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="StandbyAdvertWidget" name="StandbyADPage"/>
     <widget class="SaleAdvertWidget" name="SaleADPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StandbyAdvertWidget</class>
   <extends>QWidget</extends>
   <header>StandbyAdvertWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SaleAdvertWidget</class>
   <extends>QWidget</extends>
   <header>SaleAdvertWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
